wpilibNewCommands/src/main/native/cpp/frc2/command/CommandPtr.cpp

```diff
@@ -165,6 +165,16 @@ CommandPtr CommandPtr::OnlyIf(std::function<bool()> condition) && {
   return std::move(*this).Unless(std::not_fn(std::move(condition)));
 }
 
+CommandPtr CommandPtr::After(std::function<bool()> condition) && {
+  AssertValid();
+  return std::move(*this).BeforeStarting(std::make_unique<WaitUntilCommand>(std::move(condition)));
+}
+
+CommandPtr CommandPtr::AfterTime(units::second_t duration) && {
+  AssertValid();
+  return std::move(*this).BeforeStarting(std::make_unique<WaitCommand>(duration));
```

Comment:
```suggestion
  return std::move(*this).BeforeStarting(cmd::Wait(duration));
```
(See above)
And again, alternate version using the constructor (This is exactly how `frc::cmd::Wait()` and `frc::cmd::WaitUntil()` are implemented):
```suggestion
  return std::move(*this).BeforeStarting(WaitCommand(duration).ToPtr());
```